store/sqlstore/team_store.go

```diff
@@ -361,16 +361,26 @@ func (s SqlTeamStore) GetAll() ([]*model.Team, *model.AppError) {
 	return teams, nil
 }
 
-func (s SqlTeamStore) GetAllPage(offset int, limit int) store.StoreChannel {
-	return store.Do(func(result *store.StoreResult) {
-		var data []*model.Team
-		if _, err := s.GetReplica().Select(&data, "SELECT * FROM Teams ORDER BY DisplayName LIMIT :Limit OFFSET :Offset", map[string]interface{}{"Offset": offset, "Limit": limit}); err != nil {
-			result.Err = model.NewAppError("SqlTeamStore.GetAllTeams", "store.sql_team.get_all.app_error", nil, err.Error(), http.StatusInternalServerError)
-			return
-		}
+func (s SqlTeamStore) GetAllPage(offset int, limit int) ([]*model.Team, *model.AppError) {
+	var teams []*model.Team
+
+	if _, err := s.GetReplica().Select(&teams,
+		`SELECT
+			*
+		FROM
+			Teams
+		ORDER BY
+			DisplayName
+		LIMIT
+			:Limit
+		OFFSET
+			:Offset`, map[string]interface{}{"Offset": offset, "Limit": limit}); err != nil {
+		return nil, model.NewAppError("SqlTeamStore.GetAllTeams",
+			"store.sql_team.get_all.app_error", nil, err.Error(), http.StatusInternalServerError)
+	} else {
```

Comment:
Because the if clause ends with a `return` you can just remove the `else` clause and directly `return teams, nil`